File: Server/ServerProject/src/main/java/com/dani/objectXml/Player.java
package com.dani.objectXml;

public class Player {
    private Integer posX;
    private Integer posY;

    public Player() {
    }

    public Player(Integer posX, Integer posY) {
        this.posX = posX;
        this.posY = posY;
    }

    public Integer getPosX() {
        return posX;
    }

    public void setPosX(Integer posX) {
        this.posX = posX;
    }

    public Integer getPosY() {
        return posY;
    }

    public void setPosY(Integer posY) {
        this.posY = posY;
    }

    @Override
    public String toString() {
        return "Player{" +
                "posX=" + posX +
                ", posY=" + posY +
                '}';
    }
}

File: AppMovil/app/src/main/java/com/dani/appmovil/objectsWorld/Player.kt
package com.dani.appmovil.objectsWorld

import java.io.Serializable

data class Player(var posX:Integer?, var posY:Integer?):
    Serializable {
    constructor():this(Integer(0), Integer(0))
    override fun toString(): String {
        return "Player(posX='$posX', posY='$posY')"
    }
    fun clean() {
        posX = null
        posY = null
    }
}